dora/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -114,6 +114,7 @@ public class LocalCacheManager implements CacheManager {
    */
   private final AtomicReference<CacheManager.State> mState = new AtomicReference<>();
   private final CacheManagerOptions mOptions;
+  private Optional<Predicate<PageInfo>> mPagePredicate = Optional.empty();
```

Comment:
nit: mark final whenever possible
```suggestion
  private final Optional<Predicate<PageInfo>> mPagePredicate = Optional.empty();
```

Reply:
done